README.md

```diff
@@ -14,13 +14,26 @@ Kibana is an open source ([Apache Licensed](https://github.com/elastic/kibana/bl
 * Run `bin/kibana` on unix, or `bin\kibana.bat` on Windows.
 * Visit [http://localhost:5601](http://localhost:5601)
 
-
 ## Upgrade from previous version
 
 * Move any custom configurations in your old kibana.yml to your new one
 * Reinstall plugins
 * Start or restart Kibana
 
+## Version compatibility with Elasticsearch
+
+Ideally, you should be running Elasticsearch and Kibana with matching version numbers (💚 in the table below). If your Elasticsearch has an older version number or a newer _major_ number than Kibana, then Kibana will fail to run (🚫). If Elasticsearch has a newer minor or patch number than Kibana, then the Kibana Server will log a warning (⚠️).
+
+| Kibana version | ES version | Outcome | Description |
+| -------------- | ---------- | ------- | ----------- |
+| 6.1.2          | 6.1.2      | 💚      | Versions are the same. |
+| 6.1.2          | 6.1.3      | ⚠️      | ES patch number is newer. |
+| 6.1.2          | 6.2.0      | ⚠️      | ES major number is newer. |
+| 6.1.2          | 7.0.0      | 🚫      | ES major number is newer. |
```

Comment:
@pickypg pointed out that we don't illustrate running from `6.1` against something like `6.5`, which would be supported  right?